source/core/parameters_lagrangian.cc

```diff
@@ -882,13 +882,14 @@ namespace Parameters
           "dmt cut-off threshold",
           "0.1",
           Patterns::Double(),
-          "Cut-off threshold above which the Van der Waal forces are ignored for the DMT model relative to the pull-off force");
+          "Cut-off threshold above which the Van der Waal forces are "
+          "ignored for the DMT model relative to the pull-off force");
 
         prm.declare_entry(
           "rolling resistance torque method",
-          "constant_resistance",
+          "constant",
           Patterns::Selection(
-            "no_resistance|constant_resistance|viscous_resistance|epsd_resistance"),
+            "none|no_resistance|constant|constant_resistance|viscous|viscous_resistance|epsd|epsd_resistance"),
```

Comment:
```suggestion
            "none|no_resistance|constant|constant_resistance|viscous|viscous_resistance|epsd|epsd_resistance"),
```

Ahh wow ok, you marked everything for deprecation. That's pretty dope.
We will deprecate these parameter after the next numbered release.

Reply:
Maybe we should keep a track of what we have currently deprecated, but I don't think it's that big of a deal.

Reply:
I don't know how we should do that... 

We could change the declare_entry function in dealii? We could add an input parameter, to the declare-entry function... something like const std::string &deprecated_entries = "" ? If the patterns is a selection, it checks if the entry is deprecated and writes a warning message if true. 

When we launch a new release, we would just have to set those strings to back to ""